webapp/lib/charts.dart

```diff
@@ -497,6 +497,8 @@ class DriverTimeseriesBarChartView {
       timeScaleOptions.unit = 'minute';
       timeScaleOptions.displayFormats = new chartjs.TimeDisplayFormat(hour: 'D/MM h:mm a');
       timeScaleOptions.stepSize = 15;
+    } else if (xUpperLimit.difference(xLowerLimit).inHours == 5 || xUpperLimit.difference(xLowerLimit).inHours == 11) {
```

Comment:
I think this would generalise a bit better if we can turn this into a `<= 12` comparison like we do for under 3 hours?
```suggestion
    } else if (xUpperLimit.difference(xLowerLimit).inHours <= 12) {
```

Reply:
Good suggestion. Thanks